consensus/abci.go

```diff
@@ -94,127 +89,110 @@ func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.Resp
 
 // SetOption sets a configuration option
 func (abci *TendermintABCI) SetOption(req tmtAbciTypes.RequestSetOption) tmtAbciTypes.ResponseSetOption {
-	abci.logger.Info("TendermintABCI::SetOption()")
 	return tmtAbciTypes.ResponseSetOption{Code: tmtAbciTypes.CodeTypeOK, Log: ""}
 }
 
 // InitChain initializes the validator set
 func (abci *TendermintABCI) InitChain(req tmtAbciTypes.RequestInitChain) tmtAbciTypes.ResponseInitChain {
-	abci.logger.Info("TendermintABCI::InitChain()")
 	return tmtAbciTypes.ResponseInitChain{}
 }
 
 // CheckTx checks a transaction is valid but does not mutate the state
 func (abci *TendermintABCI) CheckTx(txBytes []byte) tmtAbciTypes.ResponseCheckTx {
-	abci.logger.Info("TendermintABCI::CheckTx()")
```

Comment:
Restore as `Debug()`

Reply:
I removed debug logs from empty methods, there is no point because there is nothing to debug.

Reply:
Same as above